orchagent/port.h

```diff
@@ -17,6 +18,7 @@ extern "C" {
  * hence setting to 1492 (1514 - 22)
  */
 #define DEFAULT_MTU             1492
+#define MAX_DEPENDENCY          20 
```

Comment:
Any specific reason for 20?

Reply:
I am gonna delete this. Its unused for now!